README.md

```diff
@@ -1 +1,56 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard project 1
+
+## Description
+This is a basic website; a Leader board Application that displays a list that looks and behaves like the part of minimalist project. 
+
+
+## Getting started
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
+
+Download the ZIP from this location, or run the following git command to clone the files to your machine:
+
+* Once the files are on your machine, open the Leaderboard folder in Visual Studio Code.
+
+* With the files open in Visual Studio Code, press the Go Live button at the bottom of the window to launch the files with Live Server.
+
+## How it works
+
+The Leaderboard web application allows the user to enter the name and the score of a leader.
+* By clicking on the submit button the information regarding the leader is stored in memory;
+* By clicking on the refresh button the list of leader in memory and their score are displayed on the screen;
+
+## Technologies Used
+* CSS
+* HTML
+* JAVASCRIPT
+* GitHub
+
+## Author
+
+#### 👤 Author
+- GitHub: [@david-lafontant](https://github.com/david-lafontant)
+- Twitter: [@manikatex](https://twitter.com/manikatex)
+
+## Contributing 
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+Please give a ⭐️ if you like this project! 
+
+## Acknowledgments
+- Hat tip to anyone contributed one way or the other.
+- Inspiration
+- etc
+
+
+## Demo link
+
+[Demo Link](https://david-lafontant.github.io/Leaderboard/)
+
+## Screenshot
+
+![Screenshot](screenshot.png)
+
```

Comment:
- [ ] [OPTIONAL] I would advise putting the screenshot of your application and the live demo at the top of your readme file.

Reply:
Asked change implemented